freeciv-web/src/main/webapp/javascript/player.js

```diff
@@ -247,3 +247,20 @@ function player_has_wonder(playerno, improvement_id)
   }
   return false;
 }
+
+/**************************************************************************
+ returns the the player's capital city, or undefined
+ **************************************************************************/
+function player_capital(player)
+{
+  return Object.values(cities)
+    .find(city => does_player_own_city(player, city) && is_capital(city));
```

Comment:
Can't use Object.values nor Array.find. Our friend MSIE again(*), and this time they are not translated in compilation.
This one is still readable even if more verbose in old JS. Or maybe we should start adding polyfills and forget these things once and for all.

(*) No, I haven't tested. Does one of the libraries already provide a definition for those functions?

Reply:
Something _does_ polyfill it\*, but I'm still searching for what might be doing so.

I've reverted to the `for...in` construct in common use, leaving the polyfill question for other issues.

\* From an IE console running freeciv-web `Object.values` is defined and works as expected, while not so in a default window console.